spec/services/idv/profile_maker_spec.rb

```diff
@@ -84,9 +87,26 @@
         expect(profile.initiating_service_provider).to eq(nil)
         expect(profile.verified_at).to be_nil
       end
+
       it 'marks the profile as legacy_unsupervised' do
         expect(profile.idv_level).to eql('legacy_unsupervised')
       end
+
+      context 'when GPO verification is needed' do
+        let(:gpo_verification_needed) { true }
+
+        it 'is not fraud_review_pending?' do
+          expect(profile.fraud_review_pending?).to eq(false)
+        end
+      end
+
+      context 'when IPP is needed' do
+        let(:in_person_verification_needed) { true }
+
+        it 'is not fraud_review_pending?' do
+          expect(profile.fraud_review_pending?).to eq(false)
+        end
+      end
```

Comment:
These test my change to the ProfileMaker. Without the change, but with this test, the GPO verification case passes, but the IPP one did not.